src/main/topLevelPages/myDashboard/newVirtualContributorWizard/newVirtualContributorQueries.graphql

```diff
@@ -0,0 +1,26 @@
+query NewVirtualContributorMySpaces {
+  me {
+    id
+    mySpaces {
```

Comment:
Add here the fitler:
mySpaces(showOnlyMyCreatedSpaces: true) 

That would ensure that you get only your spaces.